SecretsManagerRDSPostgreSQLRotationMultiUser/lambda_function.py

```diff
@@ -192,6 +192,10 @@ def set_secret(service_client, arn, token):
                 create_role = "CREATE ROLE \"%s\"" % pending_dict['username']
                 cur.execute(create_role + " WITH LOGIN PASSWORD %s", (pending_dict['password'],))
                 cur.execute("GRANT \"%s\" TO \"%s\"" % (current_dict['username'], pending_dict['username']))
+
+                # Set second user to assume the first role on login
+                # Created objects can be shared in between the two user roles in this way
+                cur.execute("ALTER USER \"%s\" SET ROLE \"%s\"" % (pending_dict['username'], current_dict['username']))
```

Comment:
Actually using user1 as the shared role has the flaw that user2 will be able to change user1's password.

In order to have users that can't change their passwords as the some documentation suggests, I believe there needs to be a shared role in this case:
https://docs.aws.amazon.com/secretsmanager/latest/userguide/enable-rotation-rds.html
> Multi-user rotation: A rotation function for a secret that stores credentials for a user that can't change their password.